runtime/pack/dist/opt/comment/plugin/comment.vim

```diff
@@ -5,9 +5,16 @@ vim9script
 
 import autoload 'comment.vim'
 
-nnoremap <silent> <expr> gc comment.Toggle()
-xnoremap <silent> <expr> gc comment.Toggle()
-nnoremap <silent> <expr> gcc comment.Toggle() .. '_'
+nnoremap <silent> <expr> <Plug>(comment-toggle) comment.Toggle()
+xnoremap <silent> <expr> <Plug>(comment-toggle) comment.Toggle()
+nnoremap <silent> <expr> <Plug>(comment-toggle-line) comment.Toggle() .. '_'
+
+# Note: use <Plug> mappings as <expr> mappings relying on imports cannot be
+# evaluated outside of this script file, e.g. from a plugin like vim-which-key
+# The <Plug> mappings also benefit from reading more like a description
+nnoremap gc <Plug>(comment-toggle)
```

Comment:
Are you sure `*noremap` would work here? I might remember it incorrectly, but `<Plug>`s should be using `*map`?

Reply:
`*noremap` works, but you are right, I should be using `*map` (`<Plug>` always uses remap since Patch 8.2.4498), will update